src/applications/vaos/tests/new-appointment/components/VAFacilityPage/index.single-site.unit.spec.jsx

```diff
@@ -56,870 +56,6 @@ describe('VAOS integration: VA facility page with a single-site user', () => {
   beforeEach(() => mockFetch());
   afterEach(() => resetFetch());
 
-  it('should show form with single required facility question', async () => {
```

Comment:
These tests are being removed because now that we no longer have the flat facility page flags, the only way to get to the page is by being registered at a Cerner site. If you're only registered at one site and it's a Cerner site, you can't do anything on the facility page, you just have one disabled option.